### PR TITLE
Volume of Torus

### PR DESCRIPTION
**Pull request by ChrisO345**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:
- Added volume of torus algorithm to Volume.js

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

<a href="https://gitpod.io/#https://github.com/TheAlgorithms/JavaScript/pull/1220"></a>

### CODE REVIEW

File: Maths/Volume.js [hunk 1]
```diff
@@ -112,6 +113,20 @@ const volHemisphere = (radius) => {
   return (2.0 * Math.PI * radius ** 3) / 3.0
 }
 
+/*
+  Calculate the volume for a Torus
+  Reference: https://en.wikipedia.org/wiki/Torus
+  return (PI * innerRadius^2) * (2 * PI * outerRadius)
```
Comment (appgurueu) [inline Maths/Volume.js:119]: Please consistently use `inner`/`outer` radius (you're calling it `minor`/`major` below)

File: Maths/Volume.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ const volHemisphere = (radius) => {
 /*
   Calculate the volume for a Torus
   Reference: https://en.wikipedia.org/wiki/Torus
-  return (PI * innerRadius^2) * (2 * PI * outerRadius)
+  return (PI ** 2 * 2 * majorRadius * minorRadius ** 2)
```
Comment (appgurueu) [inline Maths/Volume.js:119]: now that's just redundant...

Comment (ChrisO345) [reply Maths/Volume.js:119]: In what way?

Comment (appgurueu) [reply Maths/Volume.js:119]: It's redundant with the below code

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please

1. Add at least one test
2. Use a proper JSDoc comment

**Pull request closed without merging**